index.html

```diff
@@ -1,327 +1,339 @@
+        </ul>
+      </div>
+      <button class="btn sub-card-btn" type="button">See Project</button>
+    </div>
+    <!-- Card -->
+    <div class="sub-card">
+      <div class="sub-card-div">
+        <h2 class="work-card-h2">Profesional Art Printing Data</h2>
+        <p class="work-card-p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="work-card-ul">
+          <li class="work-card-li">html</li>
+          <li class="work-card-li">bootstrap</li>
+          <li class="work-card-li">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn sub-card-btn" type="button">See Project</button>
+    </div>
+    <!-- Card -->
+    <div class="sub-card">
+      <div class="sub-card-div">
+        <h2 class="work-card-h2">Profesional Art Printing Data</h2>
+        <p class="work-card-p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="work-card-ul">
+          <li class="work-card-li">html</li>
+          <li class="work-card-li">bootstrap</li>
+          <li class="work-card-li">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn sub-card-btn" type="button">See Project</button>
+    </div>
+  </div>
+</section>
+
+<!-- About Me -->
+<section class="about-section">
+  <div class="about-div">
+    <h1>About me</h1>
+    <p>
+      Hello I'm a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience! If you
+      like what you see and have a project you need coded, don't hestiate to
+      contact me.
+    </p>
+    <div class="about-btn-div">
+      <button class="btn about-btn" type="button">Get My Resume</button>
+    </div>
+  </div>
+  <div class="about-me-underline">
+    <hr />
+  </div>
+
+  <div class="about-me-techs">
+    <ul>
+      <li>
+        <div class="first-li-item">
+          <div class="diamond"></div>
+          <h3>Language</h3>
         </div>
-        <div class="icon">
-          <ul>
-            <li>
-              <a href="#"><img src="./images/img1.png" alt="Github-icon" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="./images/img2.png" alt="LinkeIn-icon" /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img src="./images/img3.png" alt="AngelList-icon"
-              /></a>
-            </li>
-            <li>
-              <a href="#"><img src="./images/img4.png" alt="Twitter-icon" /></a>
-            </li>
-            <li>
-              <a href="#"><img src="./images/img5.png" alt="Medium-icon" /></a>
-            </li>
-          </ul>
+      </li>
+      <li>Javascript</li>
+      <li>Ruby</li>
+      <li>Html</li>
+      <li>Css</li>
+    </ul>
+    <ul>
+      <li>
+        <div class="first-li-item">
```

Comment:
[Optional] I suggest that you implement that alignment of the skills section listings to match the Figma design. Check the above comments for reasons you should consider sticking to designs as closely as possible.

Your design:
![image](https://user-images.githubusercontent.com/20554583/187043902-6e2a1e11-bca3-43bd-839a-36be6aa0af0e.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187043885-07886520-a7ee-4b91-a0ef-82e8f6173617.png)